internal/security/secretstoreclient/vault.go

```diff
@@ -80,9 +47,8 @@ func NewSecretStoreClient(logger logger.LoggingClient, r internal.HttpCaller, s
 }
 
 func (vc *vaultClient) HealthCheck() (statusCode int, err error) {
-	url := vc.buildURL(VaultHealthAPI)
 	empty := struct{}{}
-	resp, err := vc.commonRequest(nil, http.MethodGet, url, &empty)
+	resp, err := vc.commonJSONRequest(nil, http.MethodGet, VaultHealthAPI, &empty)
```

Comment:
super minor nitpick, but since this empty struct is only used here, could you roll it into it's own line? just a stylistic thing

Reply:
You mean like `resp, err := vc.commonJSONRequest(nil, http.MethodGet, VaultHealthAPI, &struct{}{})` ?

Reply:
yes

Reply:
Addressed in #1855